zapisy/apps/grade/poll/assets/components/GradePollList.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div class="accordion" id="course-sections">
+    <div v-if="isSuperuser">
+      <input
+        type="checkbox"
+        v-model="showOnlyMyCourses"
+        @change="updateCurrentList"
+      />
+      <label>Show only my courses:</label>
```

Comment:
System zapisów jest i w dającej się przewidzieć przyszłości pozostanie jednojęzyczny.
```suggestion
      <label>Pokaż tylko ankiety dotyczące moich grup i przedmiotów</label>
```